undirected.go

```diff
@@ -170,6 +170,20 @@ func (u *undirected[K, T]) Clone() (Graph[K, T], error) {
 	}, nil
 }
 
+func (u *undirected[K, T]) Order() int {
+	return len(u.vertices)
+}
+
+func (u *undirected[K, T]) Size() int {
+	size := 0
+	for _, outEdges := range u.outEdges {
+		size += len(outEdges)
+	}
+
+	// divide by 2 since every add edge operation on undirected graph is counted twice.
```

Comment:
So this is super nit-picky, but the linter rules might be a bit stricter in the future and this might be remarked.

```suggestion
	// Divide by 2 since every add edge operation on undirected graph is counted twice.
```